engines/python/setup/djl_python/huggingface.py

```diff
@@ -85,17 +75,8 @@ def enable_flash():
     return False
 
 
-def get_rolling_batch_class_from_str(rolling_batch_type: str, is_mpi: bool,
-                                     model_config):
-    if rolling_batch_type == "auto":
-        architecture = model_config.architectures[0]
-        if architecture in LMI_DIST_ADV_MODEL and is_mpi:
-            from djl_python.rolling_batch.lmi_dist_rolling_batch import LmiDistRollingBatch
-            return LmiDistRollingBatch
-        else:
-            from djl_python.rolling_batch.scheduler_rolling_batch import SchedulerRollingBatch
-            return SchedulerRollingBatch
-    elif rolling_batch_type == "scheduler":
+def get_rolling_batch_class_from_str(rolling_batch_type: str):
```

Comment:
we should have deprecation warning or a fallback for auto. There are still some customer setting that to auto today.

Reply:
"auto" is still valid, but we just handle it here instead https://github.com/deepjavalibrary/djl-serving/blob/master/wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java#L81.

I don't think the logic here is ever invoked for "auto" in the current state.